src/cdk-experimental/dialog/dialog-ref.ts

```diff
@@ -127,22 +127,49 @@ export class DialogRef<T, R = any> {
   }
 
   /** Gets an observable that emits when dialog begins opening. */
-  beforeOpen(): Observable<void> {
+  beforeOpened(): Observable<void> {
     return this._containerInstance._beforeEnter.asObservable();
   }
 
   /** Gets an observable that emits when dialog is finished opening. */
-  afterOpen(): Observable<void> {
+  afterOpened(): Observable<void> {
     return this._containerInstance._afterEnter.asObservable();
   }
 
   /** Gets an observable that emits when dialog begins closing. */
-  beforeClose(): Observable<R | undefined> {
+  beforeClosed(): Observable<R | undefined> {
     return this._containerInstance._beforeExit.pipe(map(() => this._result));
   }
 
   /** Gets an observable that emits when dialog is finished closing. */
   afterClosed(): Observable<R | undefined> {
     return this._containerInstance._afterExit.pipe(map(() => this._result));
   }
+
+  /**
+   * Gets an observable that emits when dialog is finished opening.
+   * @deprecated Use `afterOpened` instead.
+   * @deletion-target 7.0.0
+   */
+  afterOpen(): Observable<void> {
```

Comment:
Since this is in cdk-experimental we do not have to go through the deprecation process, we can just change the name.